core/src/main/java/org/jboss/pnc/causeway/brewclient/BuildTranslatorImpl.java

```diff
@@ -48,14 +46,18 @@
 
     @Override
     public KojiImport translate(BrewNVR nvr, BuildRecordRest build, BuildArtifacts artifacts) throws CausewayException {
+        String[] splittedName = nvr.getName().split("-");
```

Comment:
I think you shuld use ``split("-",2)``
Now if we have:
G: foo.bar
A: foo-bar-stuff
This will produce name foo.bar-foo-bar-stuff and this code will split it to ["foo.bar", "foo", "bar", "stuff"].